lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -24,10 +24,13 @@
     ErrorCode,
 )
 from execution_engine2.utils.Condor import Condor
-from execution_engine2.utils.MongoUtil import MongoUtil
+from execution_engine2.db.MongoUtil import MongoUtil
 from installed_clients.CatalogClient import Catalog
 from installed_clients.WorkspaceClient import Workspace
 from installed_clients.authclient import KBaseAuth
+from .utils.auth import AuthUtil
```

Comment:
Can we do full paths here for easier imports for various different testing libraries?

Reply:
Sure, fixed.